qmail-pop3d.c

```diff
@@ -63,6 +67,11 @@ void err(s) char *s;
 
 void die_nomem() { err("out of memory"); die(); }
 void die_nomaildir() { err("this user has no $HOME/Maildir"); die(); }
+void die_root() {
+  substdio_puts(&sserr,"qmail-pop3d invoked as uid 0, terminating\n");
```

Comment:
You could use ```substdio_putflush()```, which can do this with a single call. Since it's basically unused you can then reduce the size of sserrbuf, or even completely remove it.